test/beast/http/parser.cpp

```diff
@@ -366,6 +489,7 @@ class parser_test
         testGotSome();
         testIssue818();
         testIssue1187();
+        testMultipleContentLengthIssue1880();
```

Comment:
What's wrong with `testIssue1880()` like all the others?

Reply:
Does it really matter? I have no idea what issue 818 is about. Whoever comes after me will at least know what is being tested, even if they don't look in github (which may be out of business by then)

Reply:
Yes it matters because it creates noise. If you want to explain what the test does, that's fine - put it in a comment. And I think, you should probably add a link to the relevant HTTP RFC section in the implementation, as a comment.